### ISSUE TITLE
Windows builds on AppVeyor are failing

### ISSUE THREAD

**Issue by ausi** [contao/core-bundle#1061]
`FilePickerProviderTest::testGetDcaAttributes` fails since 874076de55cbf962fd5e41c00e853e4dec8842a6

`InstallCommandTest::testInstallation` fails since c38e9fc28952c88cee324a3ee5a9d3a208cd0e1f

**Comment by patrickjDE** [comment]
`InstallCommandTest::testInstallation` probably fails because of the known(?) limitation that creating symlinks via php on Windows requires administrator privileges.

**Comment by fritzmg** [comment]
That's the result of FilePickerProviderTest on my system:
```
$ vendor/bin/phpunit --filter=FilePickerProviderTest
PHPUnit 5.7.21 by Sebastian Bergmann and contributors.

Testing Contao Core Bundle
..........F.                                                      12 / 12 (100%)

Time: 374 ms, Memory: 12.00MB

There was 1 failure:

1) Contao\CoreBundle\Tests\Picker\FilePickerProviderTest::testReturnsTheDcaAttributes
Failed asserting that Array &0 (
    'fieldType' => 'radio'
    'filesOnly' => true
    'value' => 'C%3A%5CUsers%5CSpooky%5CDocuments%5CVCS%5Ccontao-core-bundle%5Ctests%5CPicker/foobar'
) is identical to Array &0 (
    'fieldType' => 'radio'
    'filesOnly' => true
    'value' => 'C:\Users\Spooky\Documents\VCS\contao-core-bundle\tests\Picker/foobar'
).

C:\Users\Spooky\Documents\VCS\contao-core-bundle\tests\Picker\FilePickerProviderTest.php:329

FAILURES!
Tests: 12, Assertions: 27, Failures: 1.
```
But this is a general problem with the test I think. `FilePickerProvider::getDcaAttributes` returns the value url encoded, while the test simply asserts `__DIR__.'/foobar'`, which is not url encoded.

**Comment by fritzmg** [comment]
> InstallCommandTest::testInstallation probably fails because of the known(?) limitation that creating symlinks via php on Windows requires administrator privileges.

No, the error happens regardless of that. Haven found the reason yet though.

**Comment by fritzmg** [comment]
Analysis of `InstallCommandTest::testInstallation`:
```php
$this->getRootDir()
```
within the test cases will return
```
C:\Users\User\Documents\VCS\contao-core-bundle\tests\Fixtures
```
for example. Then, when
```php
SymlinkUtil::symlink(
    'vendor/contao/core-bundle/src/Resources/contao/config/tcpdf.php',
    'system/config/tcpdf.php',
    $this->rootDir
);
```
is executed, it tries to symlink from
```
C:\Users\User\Documents\VCS\contao-core-bundle\tests\Fixtures\vendor\contao\core-bundle\src\Resources\contao\config\tcpdf.php
```
to
```
C:\Users\User\Documents\VCS\contao-core-bundle\tests\Fixtures\system\config\tcpdf.php
```
which of course fails, because the former does not exist. The correct path would be
```
C:\Users\User\Documents\VCS\contao-core-bundle\src\Resources\contao\config\tcpdf.php
```
This test should also fail on all systems, not just Windows. Though may be I am not getting something.

**Comment by ausi** [comment]
> But this is a general problem with the test I think. FilePickerProvider::getDcaAttributes returns the value url encoded, while the test simply asserts __DIR__.'/foobar', which is not url encoded. This should fail on any system.

Windows uses the backslash for paths and unix uses the slash, the slash gets probably not urlencoded which would make the difference between the systems. I will try to fix this. Thanks for taking a look!

**Comment by fritzmg** [comment]
> Windows uses the backslash for paths and unix uses the slash, the slash gets probably not urlencoded which would make the difference between the systems.

Yeah, makes sense :)

**Comment by ausi** [comment]
> This test should also fail on all systems, not just Windows. Though may be I am not getting something.

I think symlinks to non-existent targets are allowed on unix. I will fix :)

**Comment by ausi** [comment]
Both issues fixed in #1069.

Thank you @fritzmg
